deepchem/models/torch_models/layers.py

```diff
@@ -5,6 +5,7 @@
 import sympy as sym
 from typing import Any, Tuple, Optional, Sequence, List, Union, Callable, Dict, TypedDict
 from collections.abc import Sequence as SequenceCollection
+from packaging import version
```

Comment:
Can you add a section on fibers to the deepchem datastructures page? Fibers are new to deepchem so we should document them thoroughly****